embrace-android-sdk/src/test/java/io/embrace/android/embracesdk/internal/logs/LogOrchestratorTest.kt

```diff
@@ -0,0 +1,96 @@
+package io.embrace.android.embracesdk.internal.logs
+
+import io.embrace.android.embracesdk.comms.delivery.DeliveryService
+import io.embrace.android.embracesdk.concurrency.BlockingScheduledExecutorService
+import io.embrace.android.embracesdk.fakes.FakeClock
+import io.embrace.android.embracesdk.fakes.FakeLogRecordData
+import io.embrace.android.embracesdk.worker.ScheduledWorker
+import io.mockk.mockk
+import io.mockk.verify
+import io.opentelemetry.sdk.logs.data.LogRecordData
+import org.junit.Assert.assertEquals
+import org.junit.Assert.assertFalse
+import org.junit.Assert.assertTrue
+import org.junit.Before
+import org.junit.Test
+
+internal class LogOrchestratorTest {
+
+    companion object {
+        private const val now = 123L
+    }
+
+    private lateinit var logOrchestrator: LogOrchestrator
+
+    private lateinit var executorService: BlockingScheduledExecutorService
+    private lateinit var scheduledWorker: ScheduledWorker
+    private val clock = FakeClock()
+    private val logSink: LogSinkImpl = LogSinkImpl()
+    private val deliveryService: DeliveryService = mockk(relaxed = true)
```

Comment:
You can use a FakeDeliveryService here and check the actual payload that gets delivered rather than just that a mocked method is invoked.

Reply:
Switched to FakeDeliveryService